ecchronos-binary/src/bin/ecctool.py

```diff
@@ -336,8 +445,12 @@ def get_jvm_opts(conf_dir):
 def run_ecc(cwd, command, arguments):
     if arguments.foreground:
         command += " -f"
-    proc = subprocess.Popen(command.split(" "), stdout=subprocess.PIPE, stderr=subprocess.STDOUT, # pylint: disable=consider-using-with
-                            cwd=cwd)
+    proc = subprocess.Popen(
+        command.split(" "),
+        stdout=subprocess.PIPE,
+        stderr=subprocess.STDOUT,  # pylint: disable=consider-using-with
```

Comment:
The comment suppressing pylint error should be on L448

Reply:
Done